recipes/metsim/meta.yaml

```diff
@@ -39,7 +39,6 @@ about:
   home: https://github.com/UW-Hydro/MetSim
   license: GPL-3.0
   license_family: GPL3
-  license_file: LICENSE
```

Comment:
B/c the license is not in the tarball, you'll need a `MANIFEST.in` upstream, you'll had to:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually